android/src/main/java/ie/gov/tracing/ExposureNotificationModule.java

```diff
@@ -84,6 +84,15 @@ public void start(Promise promise) {
         Tracing.start(promise);
     }
 
+    @ReactMethod
+    public void pause(Promise promise) {
+        if(nearbyNotSupported()){
```

Comment:
I would prefer this to throw - as per iOS if not authorised etc - as it is an error

Reply:
Will make those changes in a separate PR, none of these methods use reject when they should